src/main/java/com/nike/cerberus/command/CerberusCommand.java

```diff
@@ -126,7 +145,7 @@ public String getEnvironment() {
      * 2. If an env yaml was supplied use the region defined there
      * 3. If 1 and 2 fail look for value in CERBERUS_CLI_REGION env var
      */
-    public String getRegion() {
+    public String getConfigRegion() {
```

Comment:
Is this the primary region?  Or what is a "config region" if we have it stored in more than one region?

Reply:
Discussed verbally, can be removed in later PR